linetools/isgm/tests/test_use_abscomp.py

```diff
@@ -27,13 +27,22 @@
 #pdb.set_trace()
 # Set of Input lines
 
+class DummyObj(object):
+    pass
 
 
 def data_path(filename):
     data_dir = os.path.join(os.path.dirname(__file__), 'files')
     return os.path.join(data_dir, filename)
 
 
+def test_write():
+    # Component
+    abscomp,_ = mk_comp('SiII', vlim=[-250,80.]*u.km/u.s)
+    # Write
+    abscomp.write(data_path('tmp.json'))
+    # Read
+    tmpcomp = abscomp.from_json(data_path('tmp.json'))
 
```

Comment:
check that sensible properties in `tmpcomp` are the same than the original `abscomp`

Reply:
added a few